### PR TITLE
[more-examples] Added more examples

### PR DESCRIPTION
**Pull request by martinszeltins**
Moved examples from my repo to `regle-examples` repo

### CODE REVIEW

File: examples/conditional-rules-example
Comment (victorgarciaesgi) [inline examples/conditional-rules-example:1]: It seems there is still your old `.git` folder here! Don't know if it's wanted

Comment (martinszeltins) [reply examples/conditional-rules-example:1]: oops, forgot to remove it

Comment (martinszeltins) [reply examples/conditional-rules-example:1]: trying to figure out how to delete it 😆

Comment (victorgarciaesgi) [reply examples/conditional-rules-example:1]: Ahaha no worries!

Comment (martinszeltins) [reply examples/conditional-rules-example:1]: I tied myself up in knots.... 😄 I will just close this PR and open a new one

**Pull request closed without merging**